utils/src/ooinstall/variants.py

```diff
@@ -49,9 +49,16 @@ def latest_version(self):
     ]
 )
 
-# Ordered list of variants we can install, first is the default.
-SUPPORTED_VARIANTS = (OSE, AEP)
+origin = Variant('origin', 'OpenShift Origin',
+    [
+        Version('1.2', 'origin'),
+        Version('1.1', 'origin')
```

Comment:
Is there any value in keeping 1.1 around at this point? I believe 1.2 is the first origin release that we shipped to the CentOS channels which is the only place that I would expect to find older versions at this point.